exporters/metrics/src/test/java/com/google/cloud/opentelemetry/metric/FakeData.java

```diff
@@ -0,0 +1,48 @@
+package com.google.cloud.opentelemetry.metric;
+
+import com.google.auth.Credentials;
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+import io.opentelemetry.common.Attributes;
+import io.opentelemetry.common.Labels;
+import io.opentelemetry.sdk.common.InstrumentationLibraryInfo;
+import io.opentelemetry.sdk.metrics.data.MetricData.Descriptor;
+import io.opentelemetry.sdk.metrics.data.MetricData.Descriptor.Type;
+import io.opentelemetry.sdk.metrics.data.MetricData.LongPoint;
+import io.opentelemetry.sdk.metrics.data.MetricData.Point;
+import io.opentelemetry.sdk.resources.Resource;
+import java.util.Date;
+
+public class FakeData {
+
+  private static final long NANO_PER_SECOND = (long) 1e9;
+
+  static final String aProjectId = "TestProjectId";
+
+  static final Credentials aFakeCredential =
+      GoogleCredentials.newBuilder().setAccessToken(new AccessToken("fake", new Date(100))).build();
+
+  static Labels someLabels = Labels.newBuilder().setLabel("label1", "value1").setLabel("label2", "False").build();
+
+  static Descriptor aMonotonicLongDescriptor = Descriptor
+      .create("DescriptorName", "Descriptor description", "Unit", Type.MONOTONIC_LONG,
+          someLabels);
+
+  static Attributes someGceAttributes = Attributes.newBuilder()
+      .setAttribute("cloud.account.id", 123)
+      .setAttribute("host.id", "host")
+      .setAttribute("cloud.zone", "US")
+      .setAttribute("cloud.provider", "gcp")
+      .setAttribute("extra_info", "extra")
+      .setAttribute("gcp.resource_type", "gce_instance")
```

Comment:
We haven't decided if we want to keep this resource attribute (most likely not), could you please remove it?
Fine to keep other GCP attributes, that will be useful once we add support for translating them.